packages/react/src/input-base/InputBase.css.ts

```diff
@@ -63,8 +59,12 @@ export const input = recipe({
     {
       bg: "transparent",
       flex: "auto",
+      fontWeight: "400",
```

Comment:
400 is the default font-weight - don't think this is going to do anything, right?